l10n/en/firefox/welcome/page8.ftl

```diff
@@ -12,6 +12,11 @@ welcome-page8-your-privacy-respected = Your privacy respected, <strong>automatic
 
 welcome-page8-firefox-automatically-protects = { -brand-name-firefox } automatically protects your privacy, so you’re free to live your life every time you get online.
 welcome-page8-view-your-protection-report = View your protection report
+welcome-page8-tracker-watching-description = Keep prying eyes from tracking your every move.
```

Comment:
I'm not sure what the test plan is for this experiment just yet, but we may want to consider how and when we expose new strings in Pontoon. Adding these strings will generate work for ~100 locales, but I'm assuming the experiment is not targeting such a wide audience. We may want to consider hard-coding strings for the experiment, and only exposing the winning combination once the experiment is over?

Reply:
As noted in my comment above, I'm still waiting on translations from DE and FR. It makes sens that I should paste them in, will change.